vassal-app/src/main/java/VASSAL/build/module/PlayerRoster.java

```diff
@@ -475,10 +497,7 @@ protected String promptForSide() {
     int i = (myidx >= 0) ? ((myidx + 1) % sides.size()) : 0;   // If we do, start looking in the "next" slot, otherwise start at beginning.
     for (int tries = 0; i != myidx && tries < sides.size(); i = (i + 1) % sides.size(), tries++) { // Wrap-around search of sides
       String s = sides.get(i);
-      if (!alreadyTaken.contains(s) &&
-          !Resources.getString("PlayerRoster.solitaire").equals(s) &&
-          !Resources.getString("PlayerRoster.solo").equals(s) &&
-          !Resources.getString("PlayerRoster.referee").equals(s)) {
```

Comment:
It looks like we were translating these? How's that handled now?

Reply:
Oh I should change that to call the isSoloSide thinger. Didn't spot that one.

Reply:
Oh right, this is stuff I took out. "I" was thinking these were translated here but we weren't actually calling the translate-the-side method it is just using straight side names.

Which, now that you mention it, I think this has just "been wrong" for a long time. I will have a look.

Reply:
So confusing. In "PlayerRoster", "sides" means translated sides. But in "PlayerInfo", "side" means untranslated side. Anyway I have now set it to be used consistent with the way this stuff is apparently done.